src/Synapse/Synapse.Test/ScenarioTests/IntegrationRuntimeTests.cs

```diff
@@ -15,92 +15,73 @@
 {
     using Microsoft.WindowsAzure.Commands.ScenarioTest;
-    using ServiceManagement.Common.Models;
     using Xunit;
 
-    public class IntegrationRuntimeTests : SynapseTestBase
+    public class IntegrationRuntimeTests : SynapseTestRunner
     {
-        public XunitTracingInterceptor _logger;
-
-        public IntegrationRuntimeTests(Xunit.Abstractions.ITestOutputHelper output)
+        public IntegrationRuntimeTests(Xunit.Abstractions.ITestOutputHelper output) : base(output)
         {
-            _logger = new XunitTracingInterceptor(output);
-            XunitTracingInterceptor.AddToContext(_logger);
         }
 
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestSelfHostedIntegrationRuntime()
         {
-            SynapseTestBase.NewInstance.RunPsTest(
-                _logger,
-                "Test-SelfHosted-IntegrationRuntime");
+            TestRunner.RunTestScript("Test-SelfHosted-IntegrationRuntime");
         }
 
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestAzureIntegrationRuntime()
         {
-            string testResourceGroupName = SynapseTestBase.TestResourceGroupName;
-            if (string.IsNullOrEmpty(testResourceGroupName))
+            if (string.IsNullOrEmpty(TestResourceGroupName))
             {
-                testResourceGroupName = nameof(TestResourceGroupName);
+                TestResourceGroupName = nameof(TestResourceGroupName);
             }
 
-            string testWorkspaceName = SynapseTestBase.TestWorkspaceName;
-            if (string.IsNullOrEmpty(testWorkspaceName))
+            if (string.IsNullOrEmpty(TestWorkspaceName))
             {
-                testWorkspaceName = nameof(TestWorkspaceName);
+                TestWorkspaceName = nameof(TestWorkspaceName);
             }
 
-            SynapseTestBase.NewInstance.RunPsTest(
-                _logger,
-                "Test-Azure-IntegrationRuntime");
+            TestRunner.RunTestScript("Test-Azure-IntegrationRuntime");
         }
 
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestIntegrationRuntimePiping()
         {
-            string testResourceGroupName = SynapseTestBase.TestResourceGroupName;
-            if (string.IsNullOrEmpty(testResourceGroupName))
+            if (string.IsNullOrEmpty(TestResourceGroupName))
             {
-                testResourceGroupName = nameof(TestResourceGroupName);
+                TestResourceGroupName = nameof(TestResourceGroupName);
             }
 
-            string testWorkspaceName = SynapseTestBase.TestWorkspaceName;
-            if (string.IsNullOrEmpty(testWorkspaceName))
+            if (string.IsNullOrEmpty(TestWorkspaceName))
             {
-                testWorkspaceName = nameof(TestWorkspaceName);
+                TestWorkspaceName = nameof(TestWorkspaceName);
```

Comment:
Please confirm if this code logic is correct as you are modifying the base variable